### PR TITLE
#1823 Add a template to the clipboard when selected

### PR DESCRIPTION
**Pull request by epam-nikolai-iakushchenko**
When select a structure from template window it automatically starts to be added to canvas

### CODE REVIEW

File: packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx [hunk 1]
```diff
@@ -176,28 +176,15 @@ const TemplateDialog: FC<Props> = (props) => {
     )
   }
 
-  const result = (): Result | null => {
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx:179]: Can't we remove `Result` interface?

File: packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx [hunk 2]
```diff
@@ -318,13 +302,13 @@ export default connect(
   }),
   (dispatch: Dispatch<any>, props) => ({
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx:303]: propose to add a type for props
```
(dispatch: Dispatch<any>, props: Props) => ({
```
then you don't need to cast it to any 
```
props.onOk(tmpl)
```

**Review by Nitvex — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx (hunk 2 above)
Comment (epam-nikolai-iakushchenko) [reply packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx]: 👍 [33de419](https://github.com/epam/ketcher/pull/1892/commits/33de419369cf7716a3b04f3155a58292cedce379)

File: packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx (hunk 1 above)
Comment (epam-nikolai-iakushchenko) [reply packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx:179]: sure, thanks for the catch
[61ba157](https://github.com/epam/ketcher/pull/1892/commits/61ba157cc3b9b426ed8171bd35fcd3b7a543a8f3)

**Pull request merged**